### ISSUE TITLE
rq v1.0.2 fails to build with Rust 1.56.1 and Cargo 1.56.0

### ISSUE THREAD

**Issue by herbygillot** [dflemstr/rq#222]
Build is failing with the following:
```
error: panic message is not a string literal
   --> src/value/avro.rs:173:34
    |
173 |             Err(error) => panic!(error),
    |                                  ^^^^^
    |
note: the lint level is defined here
   --> src/lib.rs:6:9
    |
6   | #![deny(warnings)]
    |         ^^^^^^^^
    = note: `#[deny(non_fmt_panics)]` implied by `#[deny(warnings)]`
    = note: this usage of panic!() is deprecated; it will be a hard error in Rust 2021
    = note: for more information, see <https://doc.rust-lang.org/nightly/edition-guide/rust-2021/panic-macro-consistency.html>
help: add a "{}" format string to Display the message
    |
173 |             Err(error) => panic!("{}", error),
    |                                  +++++
help: or use std::panic::panic_any instead
    |
173 |             Err(error) => std::panic::panic_any(error),
    |                           ~~~~~~~~~~~~~~~~~~~~~

error: could not compile `record-query` due to previous error
```

This is on macOS Catalina 10.15.7

**Comment by alerque** [comment]
@dflemstr If I were to fix the build issues so this works on current Rust compilers would you be around to accept a PR & release? I know this hasn't seen much devel action in a while but I still find it useful and would like to keep it around. I'm trying to figure out if I'm going to need to fork to make that happen or if we can revive this repo a bit.

**Comment by alerque** [comment]
The particular issue in this thread was fixed in 95d585 some time *before* the issue was opened, so part of the problem is just that there hasn't been a release tagged since `v1.0.2`.

That being said there are a number of other warnings coming up building on 1.60, so I'll submit a PR for those shortly.

**Comment by siris** [comment]
@dflemstr Hi there, I am dev over on the Funtoo Linux project. We are seeing this exact same compile error that @herbygillot  reported but with Rust 1.63.

Is there any possible to cut a new release of this Rust crate with all the recent commits? This would prevent folks from hitting these compile errors that have integrated rq directory into Linux distribution package management systems like Funtoo Linux and are locked on your latest releases, version 1.0.2.

**Comment by dflemstr** [comment]
@siris I made a new release now with updated deps

**Comment by alerque** [comment]
This issue can be closed, v1.0.4 builds with the latest Rust toolchain.

**Comment by herbygillot** [comment]
Thank you for fixing this @dflemstr, I can confirm it's building just fine now.

I have now added `rq` to MacPorts, so one can install `rq` onto macOS in that way: https://ports.macports.org/port/rq/
